### ISSUE TITLE
Error "Validation of translation placeholders for localized (pl)..."

### ISSUE THREAD

**Issue by Wielebny666** [frenck/spook#785]
### What version of Spook are you using?

3.0.1

### What version of Home Assistant are you using?

2024.6.4

### The problem

Integration error in logs

### Anything in the logs? Paste it here!

```plain text
Rejestrator: homeassistant.helpers.translation
Źródło: helpers/translation.py:285
Pierwsze zdarzenie: 1 lipca 2024 17:45:37 (1 zdarzenia)
Ostatnio zalogowany: 1 lipca 2024 17:45:37

Validation of translation placeholders for localized (pl) string component.spook.issues.automation_unknown_area_references.description failed: ({'entity_id', 'areas', 'edit', 'automatyzacja'} != {'entity_id', 'areas', 'edit', 'automation'})
```

**Comment by Wielebny666** [comment]
https://github.com/frenck/spook/pull/786

**Comment by Wielebny666** [comment]
project is alive?

**Comment by Wielebny666** [comment]
> @Wielebny666 Sure it is

When will the MR be approved?

**Comment by frenck** [comment]
Fixed in #747, changed it via Weblate (as documented).
